decode/layout.ts

```diff
@@ -107,7 +109,7 @@ function process(node: any[] | number[], tagIndex: number): DomData {
     let hasAttribute = false;
     let attributes = {};
     let value = null;
-    let path = output.parent in hashes ? `${hashes[output.parent]}>` : null;
+    let path = output.parent in hashLookup ? `${hashes[hashLookup[output.parent]].selector}>` : null;
```

Comment:
Why is the lookup/cache needed? Wouldn't the same selector or index always return the same value? Can you add a comment that gives the answer to this?

Reply:
I simplified this and hashLookup no longer exists.